airbyte-integrations/connectors/source-databricks/acceptance-test-config.yml

```diff
@@ -0,0 +1,8 @@
+# See [Connector Acceptance Tests](https://docs.airbyte.com/connector-development/testing-connectors/connector-acceptance-tests-reference)
+# for more information about how to configure these tests
+connector_image: airbyte/source-databricks:dev
+acceptance_tests:
+  spec:
+    tests:
+      - spec_path: "src/test-integration/resources/expected_spec.json"
+        config_path: "src/test-integration/resources/dummy_config.json"
```

Comment:
@ameyabapat-bsft @itaseskii did you try to comment this part?

Reply:
commenting this part throws error
```
  source-databricks git:(source-databrick-v0) ✗ sh acceptance-test-docker.sh
[+] Building 0.3s (10/10) FINISHED                                                                                                                                                                 docker:desktop-linux
 => [internal] load build definition from Dockerfile                                                                                                                                                               0.0s
 => => transferring dockerfile: 670B                                                                                                                                                                               0.0s
 => [internal] load .dockerignore                                                                                                                                                                                  0.0s
 => => transferring context: 2B                                                                                                                                                                                    0.0s
 => [internal] load metadata for docker.io/airbyte/integration-base-java:dev                                                                                                                                       0.0s
 => [internal] load build context                                                                                                                                                                                  0.0s
 => => transferring context: 137B                                                                                                                                                                                  0.0s
 => [build 1/4] FROM docker.io/airbyte/integration-base-java:dev                                                                                                                                                   0.0s
 => CACHED [build 2/4] WORKDIR /airbyte                                                                                                                                                                            0.0s
 => CACHED [build 3/4] COPY build/distributions/source-databricks*.tar source-databricks.tar                                                                                                                       0.0s
 => CACHED [build 4/4] RUN tar xf source-databricks.tar --strip-components=1 && rm -rf source-databricks.tar                                                                                                       0.0s
 => CACHED [stage-1 3/3] COPY --from=build /airbyte /airbyte                                                                                                                                                       0.0s
 => exporting to image                                                                                                                                                                                             0.1s
 => => exporting layers                                                                                                                                                                                            0.0s
 => => writing image sha256:23fd53a0d35fc012d5849686fe9906b88c37c3a7c34b7e64e5554033f7a7ff56                                                                                                                       0.1s
 => => naming to docker.io/airbyte/source-databricks:dev                                                                                                                                                           0.0s

What's Next?
  View summary of image vulnerabilities and recommendations → docker scout quickview
latest: Pulling from airbyte/connector-acceptance-test
Digest: sha256:273f283618e92e4af6b167d5ba64c6e88cf389d199462995bff9fef64e4404a1
Status: Image is up to date for airbyte/connector-acceptance-test:latest
docker.io/airbyte/connector-acceptance-test:latest

What's Next?
  View summary of image vulnerabilities and recommendations → docker scout quickview airbyte/connector-acceptance-test:latest
============================= test session starts ==============================
platform linux -- Python 3.10.12, pytest-6.2.5, py-1.11.0, pluggy-1.2.0
rootdir: /test_input
plugins: requests-mock-1.9.3, cov-3.0.0, mock-3.6.1, timeout-1.4.2, sugar-0.9.7, anyio-3.7.1, hypothesis-6.82.3
collected 0 items / 6 errors
INTERNALERROR> Traceback (most recent call last):
INTERNALERROR>   File "/usr/local/lib/python3.10/site-packages/_pytest/main.py", line 269, in wrap_session
INTERNALERROR>     session.exitstatus = doit(config, session) or 0
INTERNALERROR>   File "/usr/local/lib/python3.10/site-packages/_pytest/main.py", line 322, in _main
INTERNALERROR>     config.hook.pytest_collection(session=session)
INTERNALERROR>   File "/usr/local/lib/python3.10/site-packages/pluggy/_hooks.py", line 433, in __call__
INTERNALERROR>     return self._hookexec(self.name, self._hookimpls, kwargs, firstresult)
INTERNALERROR>   File "/usr/local/lib/python3.10/site-packages/pluggy/_manager.py", line 112, in _hookexec
INTERNALERROR>     return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
INTERNALERROR>   File "/usr/local/lib/python3.10/site-packages/pluggy/_callers.py", line 155, in _multicall
INTERNALERROR>     return outcome.get_result()
INTERNALERROR>   File "/usr/local/lib/python3.10/site-packages/pluggy/_result.py", line 108, in get_result
INTERNALERROR>     raise exc.with_traceback(exc.__traceback__)
INTERNALERROR>   File "/usr/local/lib/python3.10/site-packages/pluggy/_callers.py", line 80, in _multicall
INTERNALERROR>     res = hook_impl.function(*args)
INTERNALERROR>   File "/usr/local/lib/python3.10/site-packages/_pytest/main.py", line 333, in pytest_collection
INTERNALERROR>     session.perform_collect()
INTERNALERROR>   File "/usr/local/lib/python3.10/site-packages/_pytest/main.py", line 637, in perform_collect
INTERNALERROR>     hook.pytest_collection_modifyitems(
INTERNALERROR>   File "/usr/local/lib/python3.10/site-packages/pluggy/_hooks.py", line 433, in __call__
INTERNALERROR>     return self._hookexec(self.name, self._hookimpls, kwargs, firstresult)
INTERNALERROR>   File "/usr/local/lib/python3.10/site-packages/pluggy/_manager.py", line 112, in _hookexec
INTERNALERROR>     return self._inner_hookexec(hook_name, methods, kwargs, firstresult)
INTERNALERROR>   File "/usr/local/lib/python3.10/site-packages/pluggy/_callers.py", line 155, in _multicall
INTERNALERROR>     return outcome.get_result()
INTERNALERROR>   File "/usr/local/lib/python3.10/site-packages/pluggy/_result.py", line 108, in get_result
INTERNALERROR>     raise exc.with_traceback(exc.__traceback__)
INTERNALERROR>   File "/usr/local/lib/python3.10/site-packages/pluggy/_callers.py", line 80, in _multicall
INTERNALERROR>     res = hook_impl.function(*args)
INTERNALERROR>   File "/app/connector_acceptance_test/plugin.py", line 120, in pytest_collection_modifyitems
INTERNALERROR>     config = load_config(config.getoption("--acceptance-test-config"))
INTERNALERROR>   File "/app/connector_acceptance_test/utils/common.py", line 38, in load_config
INTERNALERROR>     return Config.parse_obj(data)
INTERNALERROR>   File "pydantic/main.py", line 521, in pydantic.main.BaseModel.parse_obj
INTERNALERROR>   File "pydantic/main.py", line 341, in pydantic.main.BaseModel.__init__
INTERNALERROR> pydantic.error_wrappers.ValidationError: 1 validation error for Config
INTERNALERROR> acceptance_tests
INTERNALERROR>   none is not an allowed value (type=type_error.none.not_allowed)


======================== 2 warnings, 6 errors in 1.77s
```